packages/keycloak-user-management/src/components/CreateEditUser/tests/index.test.tsx

```diff
@@ -0,0 +1,209 @@
+import reducerRegistry from '@onaio/redux-reducer-registry';
+import { mount, shallow } from 'enzyme';
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { history } from '@onaio/connected-reducer-registry';
+import { Provider } from 'react-redux';
+import { Router } from 'react-router';
+import store from '../../../../../../client/src/store';
+import fetch from 'jest-fetch-mock';
+import * as keycloakUserDucks from '@opensrp/store';
+import * as fixtures from '../../../forms/tests/fixtures';
```

Comment:
Is this pulling fixtures from another package?  Please let us NOT do that.  Each package should be as independent as possible